src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -404,7 +404,7 @@ public void onOpened()
 
             if (!(request.getRequest() instanceof IDeliverable)
                   || (!isCreative && !InventoryUtils.hasItemInItemHandler(new InvWrapper(inventory),
-              stack -> ((IRequest<? extends IDeliverable>) request).getRequest().matches(stack))))
+              stack -> ((IDeliverable) request.getRequest()).matches(stack))))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1612)